lib/cn/csched_sp3.c

```diff
@@ -1141,28 +1150,18 @@ sp3_process_ingest(struct sp3 *sp)
 
     list_for_each_entry(tree, &sp->mon_tlist, ct_sched.sp3t.spt_tlink) {
         struct sp3_tree *spt = tree2spt(tree);
-        long alen, wlen;
+        long alen;
```

Comment:
This is used to read an unsinged long. Can you fix the sign?

Reply:
Yeah, I looked at that, it was troublesome.  Let me take another look...

Reply:
Ah, there no longer seems to be an underflow issue, so I'll just make it a ulong.

Reply:
If I recall correctly, the values representing per-job effect on space amp are signed because a compaction job can reduce things like "i_len" or "r_len".